### PR TITLE
Add pretzel (0.0.15)

### PR DESCRIPTION
**Pull request by CryoTheRenegade**
After making all changes to the cask:

- [ X] `brew cask audit --download {{cask_file}}` is error-free.
- [X ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X ] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [ X] Named the cask according to the [token reference].
- [X ] `brew cask install {{cask_file}}` worked successfully.
- [X ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ X] Checked there are no [open pull requests] for the same cask.
- [X ] Checked the cask was not already refused in [closed issues].
- [ X] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by jledbetter460** [comment]
Thanks, Vítor!

File: CAsks/pretzel.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'pretzel' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://download.pretzel.rocks/PretzelDesktop.dmg'
+  name 'Pretzel'
+  homepage 'https://www.pretzel.rocks/'
```
Comment (adidalal) [inline CAsks/pretzel.rb]: Also, mind adding an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

URL is `https://download.pretzel.rocks/latest-mac.json`

File: CAsks/pretzel.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'pretzel' do
+  version :latest
```
Comment (adidalal) [inline CAsks/pretzel.rb]: This can be an exact version - `0.0.15`

File: CAsks/pretzel.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  sha256 :no_check
+
+  url 'https://download.pretzel.rocks/PretzelDesktop.dmg'
```
Comment (adidalal) [inline CAsks/pretzel.rb]: url "https://download.pretzel.rocks/Pretzel-#{version}-mac.zip"

**Review by adidalal — DISMISSED** [review]

File: CAsks/pretzel.rb (hunk 1 above)
Comment (jledbetter460) [reply CAsks/pretzel.rb]: Sure thing

File: CAsks/pretzel.rb (hunk 3 above)
Comment (jledbetter460) [reply CAsks/pretzel.rb]: The installation method is a dmg not a zip

**Review by commitay — APPROVED** [review]

**Pull request merged**